chat/rpc-methods.md

```diff
@@ -50,7 +51,6 @@ Used to send a message to its peer through topic T.
 ```jsonc
 // wc_chatMessage params
 {
-  "publicKey": string,
```

Comment:
So we don't need the publicKey here because the message can be routed based on the thread topic anyways right?